### PR TITLE
[WIP] Add customImportMapDashboards plugin to manifest and add build script

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Add customImportMapDashboards to manifest and add build script. The script is provided from @kavilla https://github.com/Shivamdhar/opensearch-build/pull/2

### Issues Resolved
https://github.com/opensearch-project/dashboards-maps/issues/17

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.2.0/opensearch-dashboards-2.2.0.yml [hunk 1]
```diff
@@ -10,3 +10,7 @@ components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
     ref: 2.x
+  - name: customImportMap
+    repository: https://github.com/opensearch-project/dashboards-maps.git
+    working_directory: src/plugins/custom_import_map
+    ref: 2.2
```
Comment (Shivamdhar) [inline manifests/2.2.0/opensearch-dashboards-2.2.0.yml:16]: Hey @junqiu-lei, let's add an extra line at the end of both the files.

**Review by Shivamdhar — APPROVED** [review]
Thanks @junqiu-lei , looks good!

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]
Move it to draft as we need to test before adding it to manifest due to it is a new plugin with different structure.

**Comment by junqiu-lei** [comment]
> Move it to draft as we need to test before adding it to manifest due to it is a new plugin with different structure.

Offline synced with @peterzhuamazon and updated the plugin name to `customImportMapDashboards `

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**